Spinner/Spinner.js

```diff
@@ -127,12 +146,14 @@ const SpinnerBase = kind({
 	},
 
 	defaultProps: {
+		color: 'light',
 		size: 'medium',
 		transparent: false
 	},
 
 	styles: {
-		css: componentCss
+		css: componentCss,
+		publicClassNames: ['spinner']
```

Comment:
The `css` prop only documents `'spinner'` as a supported override so I've added it here. Without this, we would be supporting all the overrides for `ui/Spinner` since we pass `css` down to it.